src/solver/conv_winoRxS.cpp

```diff
@@ -513,17 +522,65 @@ static bool IsApplicableBase(const ConvolutionContext& params)
     }
 }
 
-bool ConvBinWinogradRxSf2x3::IsApplicable(const ConvolutionContext& params) const
+template <int Winodata, int Winofilter>
+bool ConvBinWinoRxS<Winodata, Winofilter>::IsApplicable(const ConvolutionContext& params) const
 {
-    if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_RXS_F2X3{}))
+    if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_RXS_F2X3{}) && IS2x3)
         return false;
-    return IsApplicableBase(params) && params.group_counts > 1;
+    if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_RXS_F3X2{}) && IS3x2)
+        return false;
+    if(IS3x2 && params.kernel_stride_w == 2)
+        return false;
+    auto applicable = IsApplicableBase(params);
+    if(IS2x3)
+        applicable = applicable && params.group_counts > 1;
+    return applicable;
+}
+
+template <int Winodata, int Winofilter>
+static inline boost::optional<PerformanceConfigConvBinWinogradRxS>
+GetPerfConfFromEnv(const ConvolutionContext& params)
+{
+    PerformanceConfigConvBinWinogradRxS fromEnv;
+    std::string s;
+    const char* p_asciz = nullptr;
+    const char* env_name;
+
+    if(IS2x3)
+    {
+        p_asciz  = miopen::GetStringEnv(MIOPEN_DEBUG_AMD_WINOGRAD_RXS_F2X3_PERF_VALS{});
+        env_name = MIOPEN_DEBUG_AMD_WINOGRAD_RXS_F2X3_PERF_VALS::value();
+    }
+    else if(IS3x2)
+    {
+        p_asciz  = miopen::GetStringEnv(MIOPEN_DEBUG_AMD_WINOGRAD_RXS_F3X2_PERF_VALS{});
+        env_name = MIOPEN_DEBUG_AMD_WINOGRAD_RXS_F3X2_PERF_VALS::value();
+    }
+
+    if(p_asciz == nullptr)
+        return boost::none;
```

Comment:
[Recommendation] This should construct an instance of `boost::optional<PerformanceConfigConvBinWinogradRxS>` that is empty:
```suggestion
        return {};
```